packages/repeater/src/request-runner/Response.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { Response } from './Response';
+import { Protocol } from '../models';
+
+describe('Response', () => {
+  describe('constructor', () => {
+    it('should create an instance having only protocol', () => {
+      const responseOptions = {
+        protocol: Protocol.WS
+      };
+
+      const response = new Response(responseOptions);
+
+      const { protocol, statusCode, headers, body, message, errorCode } =
+        response;
+      expect({
+        protocol,
+        statusCode,
+        headers,
+        body,
+        message,
+        errorCode
+      }).toEqual(responseOptions);
```

Comment:
Is it intended?
```suggestion
      expect(response).toEqual(responseOptions);
```

Reply:
kind of intended - as `responseOptions` is not `response` and `toEqual` check (which now is valid) will fail on introducing e. g. any new public field in `response`. i prefer to have at least `toMatchObject()` there, wdyt?

Reply:
Yeah, `toMatchObject` is a good choose